src/block/button-group/save.js

```diff
@@ -21,25 +21,33 @@ import {
 	getResponsiveClasses,
 	getRowClasses,
 } from '~stackable/block-components'
+import { applyFilters } from '@wordpress/hooks'
 
 export const Save = props => {
 	const {
 		attributes,
 	} = props
 
 	const {
+		flexWrap = '',
 		collapseOn = '',
 	} = attributes
 
 	const rowClass = getRowClasses( props.attributes )
 	const blockAlignmentClass = getAlignmentClasses( props.attributes )
 	const responsiveClass = getResponsiveClasses( props.attributes )
+	const flexWrapClass = flexWrap ? 'stk-block-button-group--flex-wrap' : flexWrap
 
-	const blockClassName = classnames( [
-		props.className,
-		'stk-block-button-group',
-		responsiveClass,
-	] )
+	const blockClassName = classnames(
+		applyFilters( 'stackable.buttonGroup.save.blockClassNames', [
```

Comment:
Please use `stackable.button-group.save.blockClassNames` as the name for the filter, the `button-group` corresponds to the name of the block `stackable/button-group`